java/src/main/java/vowpalWabbit/learner/VWLearners.java

```diff
@@ -1,8 +1,6 @@
 package vowpalWabbit.learner;
 
-import vowpalWabbit.jni.NativeUtils;
-
-import java.io.IOException;
+import java.util.Optional;
```

Comment:
Good catch. 

Optional helped to maintain cleaner and more semantically meaningful code, shall we upgrade it to Java 8 since Java 6 is no [longer updated](http://www.oracle.com/technetwork/java/eol-135779.html)?